src/components/FadeTransition.tsx

```diff
@@ -0,0 +1,35 @@
+import { ReactNode } from 'react';
+import { Transition } from 'react-transition-group';
+
+const duration = 300;
+
+const defaultStyle = {
+  transition: `opacity ${duration}ms ease-in-out`,
+  opacity: 0,
+};
+
+const transitionStyles = {
+  entering: { opacity: 1 },
+  entered: { opacity: 1 },
+  exiting: { opacity: 0 },
+  exited: { opacity: 0 },
+  unmounted: { opacity: 0 },
+};
+
+type FadeTransitionProps = {
+  in: boolean;
+  children: ReactNode;
+};
+
+export const FadeTransition = ({ in: inProp }: FadeTransitionProps) => (
+  <Transition in={inProp} timeout={duration}>
+    {(state) => (
+      <div
+        style={{
+          ...defaultStyle,
+          ...transitionStyles[state],
+        }}
+      />
+    )}
+  </Transition>
+);
```

Comment:
So, this isn't actually being used - so open question if we want to keep it in for future use.  I couldn't get it to work all the way with swapping out loading / error for data on the graphs.  In theory, this should fade in/out components based on the parent container's state.

I think a good follow-up to this PR would be to complete this for the graphs, or roll it into any future graph improvements we want to make, like animations.

Reply:
My vote would be to just remove it. Seems easy enough to add it back in if/when we add more graph improvements.